things/api/logging.go

```diff
@@ -27,180 +26,270 @@ func LoggingMiddleware(svc things.Service, logger *slog.Logger) things.Service {
 
 func (lm *loggingMiddleware) CreateThings(ctx context.Context, token string, clients ...mgclients.Client) (cs []mgclients.Client, err error) {
 	defer func(begin time.Time) {
-		message := fmt.Sprintf("Method create_things %d things using token %s took %s to complete", len(cs), token, time.Since(begin))
+		args := []interface{}{
+			slog.String("duration", time.Since(begin).String()), slog.Any("no_of_things", len(clients)),
+		}
 		if err != nil {
-			lm.logger.Warn(fmt.Sprintf("%s with error: %s.", message, err))
+			args = append(args, slog.String("error", err.Error()))
+			lm.logger.Warn("Create thing failed to complete successfully", args...)
 			return
 		}
-		lm.logger.Info(fmt.Sprintf("%s without errors.", message))
+		lm.logger.Info("Create thing completed successfully", args...)
 	}(time.Now())
 	return lm.svc.CreateThings(ctx, token, clients...)
 }
 
 func (lm *loggingMiddleware) ViewClient(ctx context.Context, token, id string) (c mgclients.Client, err error) {
 	defer func(begin time.Time) {
-		message := fmt.Sprintf("Method view_thing for thing with id %s using token %s took %s to complete", id, token, time.Since(begin))
+		args := []interface{}{
+			slog.String("duration", time.Since(begin).String()),
+			slog.String("id", id),
+		}
 		if err != nil {
-			lm.logger.Warn(fmt.Sprintf("%s with error: %s.", message, err))
+			args = append(args, slog.String("error", err.Error()))
+			lm.logger.Warn("View client failed to complete successfully", args...)
```

Comment:
```suggestion
			lm.logger.Warn("View thing failed to complete successfully", args...)
```

Do this for all cases